tests/e2e/tidbcluster/serial.go

```diff
@@ -1008,4 +1010,147 @@ var _ = ginkgo.Describe("[tidb-operator][Serial]", func() {
 			klog.Info("success to check auto scale-in tidb to 2 replicas")
 		})
 	})
+
+	ginkgo.Context("[Verify: Upgrading Operator from 1.0.6", func() {
+		var oa tests.OperatorActions
+		var ocfg *tests.OperatorConfig
+		var version string
+
+		ginkgo.BeforeEach(func() {
+			version = os.Getenv("RELEASED_VERSION")
+			if len(version) < 1 {
+				version = "v1.0.6"
+			}
```

Comment:
download v1.0.6 chart in this e2e test

actually, we can run `helm repo add pingcap https://charts.pingcap.org/ && helm install xxx --version v1.0.6` to install releases chart, but this needs some code refactoring. we can improve it in the future

Reply:
`v1.0.6` can be hard-coded in this test

Reply:
`DeployOperator` will clone the git repo if the tag is not `e2e`. If it works well, I think it's fine too.

Reply:
Actually `helm repo add` is my first choice, but it seems not stable because of the network.